README.md

```diff
@@ -27,33 +27,34 @@ A curated list of zkVM, zero-knowledge virtual machine.
 - [tutorials](#tutorials)
 - [architecture](#architecture)
-  - [precompiles](#precompiles)
-  - [recursion](#recursion)
-  - [lookup arguments](#lookup-arguments)
-  - [hardware](#hardware)
+- [precompiles](#precompiles)
+- [recursion](#recursion)
+- [lookup arguments](#lookup-arguments)
+- [hardware](#hardware)
 - [related awesome lists](#related-awesome-lists)
 
-## projects repositories
-
-- [cairo](https://github.com/lambdaclass/cairo-vm)
-- [ceno](https://github.com/scroll-tech/ceno)
-- [eigen zkvm](https://github.com/0xEigenLabs/eigen-zkvm)
-- [jolt](https://github.com/a16z/jolt)
-- [miden](https://github.com/0xPolygonMiden/miden-vm)
-- [mozak vm](https://github.com/0xmozak/mozak-vm)
-- [nexus](https://github.com/nexus-xyz/nexus-zkvm)
-- [o1vm](https://github.com/o1-labs/proof-systems/tree/master/o1vm)
-- [olavm](https://github.com/Sin7Y/olavm)
-- [powdr](https://github.com/powdr-labs/powdr)
-- [risc0](https://github.com/risc0/risc0)
-- [sp1](https://github.com/succinctlabs/sp1)
-- [sphinx](https://github.com/argumentcomputer/sphinx)
-- [triton vm](https://github.com/TritonVM/triton-vm)
-- [valida](https://github.com/valida-xyz/valida)
-- [zisk](https://github.com/0xPolygonHermez/zisk)
-- [zkLLVM](https://github.com/NilFoundation/zkLLVM)
-- [zkWasm](https://github.com/DelphinusLab/zkWasm)
-- [zkm](https://github.com/zkMIPS/zkm)
+## projects 
+
+| zkVM                                                             | ISA    | Continuations & <br> Parallel Proving | GPU            | Languages | Precompiles |
+|:----------------------------------------------------------------:|:------:|:-------------------------------------:|:--------------:|:---------:|:-----------:|
+| [cairo](https://github.com/lambdaclass/cairo-vm)                 | Cairo  |                                       |                |           |             |
+| [ceno](https://github.com/scroll-tech/ceno)                      | RISC-V |                                       |                |           |             |
+| [eigen zkvm](https://github.com/0xEigenLabs/eigen-zkvm)          | RISC-V |                                       |                |           |             |
+| [jolt](https://github.com/a16z/jolt)                             | RISC-V | :red_circle:                          |                |           |             |
+| [miden](https://github.com/0xPolygonMiden/miden-vm)              | Wasm   |                                       |                |           |             |
+| [mozak vm](https://github.com/0xmozak/mozak-vm)                  | RISC-V |                                       |                |           |             |
+| [nexus](https://github.com/nexus-xyz/nexus-zkvm)                 | RISC-V | :green_circle:                        |                |           |             |
+| [o1vm](https://github.com/o1-labs/proof-systems/tree/master/o1vm)| MIPS   |                                       |                |           |             |
```

Comment:
[olvm] prover: kimchi / language: Go (https://github.com/o1-labs/proof-systems/blob/master/o1vm/README-optimism.md) not exactly sure / GPU :x / continuation, precompile: x  (notsure also)